Library/Formula/vamp-plugin-sdk.rb

```diff
@@ -0,0 +1,75 @@
+class VampPluginSdk < Formula
+  desc "audio processing plugin system sdk"
+  homepage "http://www.vamp-plugins.org"
+  url "https://code.soundsoftware.ac.uk/attachments/download/1520/vamp-plugin-sdk-2.6.tar.gz"
+  sha256 "d0d3578137ac0c1e63f31561081a8d61da526a81152bc1dc9383b629bc07f85f"
+  head "https://code.soundsoftware.ac.uk/hg/vamp-plugin-sdk", :using => :hg
+
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libsndfile"
+  depends_on "libogg"
+  depends_on "flac"
+
+  # activate osx specific items in Makefile.in
+  # https://code.soundsoftware.ac.uk/issues/1473
+  patch :p0, :DATA
```

Comment:
Thanks for the conversation here but I think I'd rather wait until it reaches a conclusion on an agreed right approach (i.e. a patch that would be merged) before we include this. Thanks, though :heart: